go.mod

```diff
@@ -30,6 +30,6 @@ require (
 	gopkg.in/tomb.v2 v2.0.0-20161208151619-d5d1b5820637
 	gopkg.in/tylerb/graceful.v1 v1.2.15
 	gopkg.in/yaml.v2 v2.3.0
-	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
+	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
```

Comment:
You might need to remove this in order to avoid a conflict, as it was already fixed in master (https://github.com/snapcore/snapd/commit/8891cda79d088229e7c0731b8aa97afaf38e245b)